src/olympia/amo/views.py

```diff
@@ -131,3 +134,15 @@ def _frontend_view(*args, **kwargs):
 def frontend_view(*args, **kwargs):
     """Wrap _frontend_view so we can mock it in tests."""
     return _frontend_view(*args, **kwargs)
+
+
+class SiteStatusView(APIView):
+    authentication_classes = []
+    permission_classes = []
+
+    @classmethod
+    def as_view(cls, **initkwargs):
+        return non_atomic_requests(super().as_view(**initkwargs))
+
+    def get(self, request, format=None):
+        return Response(SiteStatusSerializer(object()).data)
```

Comment:
Why do we need `object()` here?

Reply:
It's just how serializers are written - SiteStatusSerializer isn't really serializing input or retrieved data in the strictest sense, and serializers are supposed to be serializing _something_. You either have to pass an instance of an object (an empty object is the most trivial example), or pass data, but then you have to call validate too (and extra few lines of code)

Reply:
ah, ok I see now, thanks